### DISCUSSION TITLE
[INFO] Disable circle center dragging

### DISCUSSION THREAD

**Discussion by ADrepo** [geoman-io/leaflet-geoman#857] [Questions & Help]
Is it possible to disable the circle center dragging? In edit mode i need to allow only radius change.

Thank you so much for the fantastic job

**Comment by Falke-Design** [comment]
You can remove the events and disable dragging of the center markers on the circles.

```
map.on('pm:globaleditmodetoggled',function (e) {
   if(e.enabled){
       map.eachLayer(function(layer){
           if(layer instanceof L.Circle && layer.pm && layer.pm._centerMarker){
               layer.pm._centerMarker.off();
               layer.pm._centerMarker.dragging.disable();
           }
       });
   }
});
```

**Comment by codeofsumit** [comment]
@ADrepo 👆 that's how to do it. For now, I would like to avoid creating a separate option for this as it adds complexity to the library that you can achieve with fairly few lines of code 👍 .
Thanks @Falke-Design for providing the solution
